front_end/panels/ai_chat/ui/AIChatPanel.ts

```
@@ -88,6 +88,10 @@ import { HelpDialog } from './HelpDialog.js';
 import { SettingsDialog, isVectorDBEnabled } from './SettingsDialog.js';
 import { EvaluationDialog } from './EvaluationDialog.js';
 import * as Snackbars from '../../../ui/components/snackbars/snackbars.js';
+// MCP integration
+import { MCPRegistry } from '../mcp/MCPRegistry.js';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)